x/ccv/provider/keeper/relay.go

```diff
@@ -60,7 +73,8 @@ func (k Keeper) OnAcknowledgementPacket(ctx sdk.Context, packet channeltypes.Pac
 			// Attempt to complete unbonding in staking module
 			err := k.stakingKeeper.UnbondingCanComplete(ctx, unbondingOp.Id)
 			if err != nil {
-				return err
+				nack := channeltypes.NewErrorAcknowledgement(err.Error())
+				return &nack
 			}
 			// Delete unbonding op
 			k.DeleteUnbondingOp(ctx, unbondingOp.Id)
```

Comment:
`DeleteUnbondingOp` is not called if `UnbondingCanComplete` returns an error. Is this intentional?